style/workLoggingSystem.css

```diff
@@ -70,56 +70,84 @@ button:hover {
 }
 
 .jobTicketCard {
-    background-color: #e65100;
+    background-color: #2196f3 ;
     padding-top: 1%;
-    padding-bottom: 2%;
+    padding-bottom: 1%;
     border-radius: 5px;
     cursor: pointer;
     color: white;
 }
 
-.jobTicketCard:hover {
-    background-color: #69f0ae;
+/*.jobTicketCard:hover {
+    background-color: #00e5ff;
 }
+*/
 
 .frameTitle {
-    background-color: #ffc107;
+   
     font-weight: 300;
 }
 
-.frameContent {
-    padding-top: 2%;
+.menuTicket {
+    background-color: #263238;
+    color: white;
+    border-top-left-radius: 5px;
+    border-bottom-left-radius: 5px;
+    font-weight: 300;
+    text-align: center;
+    cursor: pointer;
+    font-family: 'Bree Serif', serif;
+    font-size: 16px;
+    margin-left: 2%;
+    padding-top: 1%;
+    padding-bottom: 1%;
+    margin-bottom: 2%;
+    width: 40%;
+  
 }
 
 .menu {
-    background-color: #ffc107;
+    background-color: #455a64;
+    color: white;
     border-top-left-radius: 5px;
     border-bottom-left-radius: 5px;
-    font-weight: 400;
+    font-weight: 300;
     text-align: center;
     cursor: pointer;
     font-family: 'Bree Serif', serif;
-    font-size: 18px;
+    font-size: 16px;
     padding-top: 1%;
     padding-bottom: 1%;
 }
 
 .menuData {
-    background-color: #c6ff00;
+    background-color: #baffff;
     border-top-right-radius: 5px;
     border-bottom-right-radius: 5px;
     cursor: pointer;
     font-family: 'Bree Serif', serif;
     font-size: 18px;
     padding-top: 1%;
     padding-bottom: 1%;
+    width: 50%;
+    
+}
+.detail{
+    padding-top: 1%;
+    padding-bottom: 1%;
+    margin-bottom: 2%;
+    margin-left: 2%;
```

Comment:
Please try to avoid the use of margin, Try to use Padding in place of that

Reply:
okay,sure

Reply:
I have to use a margin to fix the changes I have done. It cant be done using padding.
Both have different use, margin cant be replace with the padding.